### DISCUSSION TITLE
Markdown add empty line

### DISCUSSION THREAD

**Discussion by chrisonline** [requarks/wiki#5829] [Help / Questions] [answered]
How can I add an empty line in the markdown editor?

Currently, I see:
Test
Test2
Test3

But I want to display:
Test

Test2

Test3

**Comment by NGPixel** [comment, marked as answer]
This is already the default behavior?
```
Test

Test2

Test3
```
will give you:

Test

Test2

Test3

Maybe you disabled auto line breaks in Admin > Rendering > Markdown > Core?

**Comment by chrisonline** [comment]
Thanks, this was the problem.
Now it is working.

Is it possible to add more than one empty line?

**Reply by NGPixel** [reply]
You can add 2 spaces at the end of any line to force a line break.

**Reply by chrisonline** [reply]
I know but this does not work.

**I want:**
Here you will find many infos like HowTos or other things.  
empty
empty  
test

**I get:**
![image](https://user-images.githubusercontent.com/2077218/201074771-9f018ab2-8908-40f6-bdab-7a6c1ef4aa89.png)
